homeassistant/components/androidtv_remote/config_flow.py

```diff
@@ -355,5 +358,7 @@ def _async_apps_form(self, app_id: str) -> ConfigFlowResult:
             data_schema=data_schema,
             description_placeholders={
                 "app_id": f"`{app_id}`" if app_id != APPS_NEW_ID else "",
+                "play_store_oid": PLAY_STORE_OID,
```

Comment:
Same for the constants. Your can also make them private

Reply:
Why make the constants private?

Reply:
Mostly to make it clear they aren't used anywhere else outside this file

Reply:
We could have used them in the tests, and we normally do not make these kind of variables make private.